LinqToRest/ProjectionReader.cs

```diff
@@ -1,26 +1,26 @@
-using System;
```

Comment:
Why is this file modified?

Reply:
Fixed in 379eb22827cd2968cbad74f6b4209d51eeb05254